encoding/thrift/register.go

```diff
@@ -103,10 +107,13 @@ func BuildProcedures(s Service, opts ...RegisterOption) []transport.Procedure {
 			Protocol:     proto,
 			Enveloping:   rc.Enveloping,
 		})
+		signature, _ := s.Signatures[methodName]
```

Comment:
Without the blank identifier, it will panic instead of returning an empty string if the entry cannot be found.